txbits/public/js/password-form.js

```diff
@@ -41,5 +41,21 @@ $(function(){
         }
         $password1.keyup(check_form);
         $password2.keyup(check_form);
+    }).submit(function(e) {
+        var $this = $(this);
+        var min_len = $this.attr('data-password-length');
```

Comment:
I don't see a good way to refactor it into a function. The first block needs `$password1` and `$password2` for `keyup`. The second block has different `if` conditions and `e.preventDefault();`.